src/prebid.js

```diff
@@ -441,7 +441,9 @@ $$PREBID_GLOBAL$$.renderAd = function (doc, id) {
         var url = adObject.adUrl;
         var ad = adObject.ad;
 
-        if (ad) {
+        if (doc===document) {
```

Comment:
We need mark the winning bid before bailing out if the ad render call tries to replace the top `document`. Previously I had placed it right before the `if (adObject)` check but that's too early.